dcos/dcos-ui
# DCOS-37752: Jobs / Refactor Overview UI components to use new mediator

**SCOPE**
this PR adds the final part to jobs overview refactoring - after this is merged everything should be done. 👍

**HOW TO TEST**
click trough the overview pages and make sure everything works as expected and haves exactly like master.

**TODO**
Thats all 👌

**MISC**
- ( https://jira.mesosphere.com/browse/DCOS-38850 ) there still is this annoying "delay" issue when you click around or try to filter. reason is, that the resolver fires an request and waits for it to respond before it updates the stream. instead it should just give you the last known state and update after pollingInterval. But thats out of scope for this PR IMHO
- ( https://jira.mesosphere.com/browse/DCOS-38849 ) components could get tidied up here and there
- ( https://jira.mesosphere.com/browse/DCOS-38851 ) we might want to have a `<JobInfo />`  (not really part of this PR, but it is related)
